lib/metric_fu/loader.rb

```diff
@@ -34,7 +34,7 @@ def lib_require(base = "", &_block)
     #     ::metrics_require which takes a block of files to require relative to the metrics_dir
     def create_dirs(klass)
       class << klass
-        Array(yield).each do |dir|
```

Comment:
did it just need a `create_dirs(klass, &block)` instead maybe?

Reply:
yeah, I guess that could have worked too. I thought it was a good opportunity to simplify unnecessary indirection...

Reply:
I'm not sure anyone cares, but this fix changes semantics.  One could, with the old code, override the library_dirs in use without a warning of overriding a constant (likewise for artifact_subdirs, I think).  To this fixes credit, it is simpler.  I don't know if any of these directories are subject to the whim of the tools metric_fu calls.  If they are,  then keeping the old semantics seems better.